api/src/db/dbConnection.ts

```diff
@@ -6,7 +6,7 @@ import { chainModels, getChainModels, userModels } from "@shared/dbSchemas";
 import { Template, TemplateFavorite, UserAddressName, UserSetting } from "@shared/dbSchemas/user";
 import { chainDefinitions } from "@shared/chainDefinitions";
 
-const csMap = {
+const csMap: { [key: string]: string } = {
```

Comment:
is it needed to be this flexible? if you skip this type it's implicitly typed as object with specified keys and values. It seems that what really needed here is typed env

Reply:
I removed the type and added a typeguard instead. But yeah ideally we would have env var typing & validation.